doxia-core/src/main/java/org/apache/maven/doxia/Doxia.java

```diff
@@ -52,6 +52,22 @@
     void parse( Reader source, String parserId, Sink sink )
         throws ParserNotFoundException, ParseException;
 
+    /**
+     * Parses the given source model using a parser with given id,
+     * and emits Doxia events into the given sink.
+     *
+     * @param source not null reader that provides the source document.
+     * You could use <code>newReader</code> methods from {@link org.codehaus.plexus.util.ReaderFactory}.
+     * @param parserId Identifier for the parser to use.
```

Comment:
nit: no caps and no period, per Oracle guidelines

Reply:
The exceptions in the oracle reference (https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html#@exception) do capitalize, should I correct exceptions also accordingly?

Reply:
I assume that's an oversight since it contradicts the language of the doc and only appears in an example